### DISCUSSION TITLE
upgrade oap from 9.5.0 to 10.0.0, MESH_Process_Mesh-Process conflict

### DISCUSSION THREAD

**Discussion by tunbb** [apache/skywalking#12256] [General]
my version oap 9.5.0, ui 9.3.0. I upgrade oap version to 10.0.0 and ui version to 10.0.0, ui got runing but oap crash, logs shows 
"java.lang.IllegalArgumentException: File:  mesh-process.json layer_entity_name: MESH_Process_Mesh-Process conflict with exist configuration id: d04xxxxx" , how can I fix it?

I try to delete mesh-process.json file but still the error occur

**Comment by wu-sheng** [comment]
We don't support upgrade. You need a new backend cluster and new database.

**Reply by tunbb** [reply]
how about my history data?

**Reply by wu-sheng** [reply]
No support for that.

**Reply by tunbb** [reply]
thanks~
